### PR TITLE
release-1.19, fix latest and deprecated parameters

### PR DESCRIPTION
**Pull request by kbhawkey**
Update `latest` parameter to 1.19.
Set deprecated to true.

### CODE REVIEW

**Comment by kbhawkey** [comment]
Related to #25548.

**Comment by tengqm** [comment]
Looks to me the meaning of `latest` is always relative to where you stand. The param `latest` is used in several shortcodes where the semantics are always "the current". I'm unaware of any cases we are using `latest` for other purpose. When I'm interpreting "latest" as "current", this fix looks to me.

/lgtm

**Comment by tengqm** [comment]
/lgtm cancel

A second thought on this. Maybe we should use the `version` parameter instead. Let `latest` mean *latest*.

**Comment by kbhawkey** [comment]
Yes, see my comment in PR #25552 .
I agree that latest parameter is relative to the version. I looked back in the history (release-1.14).

**Comment by kbhawkey** [comment]
How does a deprecated version use the configuration for the next version (the current/latest version)?

**Comment by kbhawkey** [comment]
The latest-version shortcode is used to generate the API reference link in a number of files.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-25553--k8s-v1-19.netlify.app/docs/concepts/architecture/nodes/

**Comment by tengqm** [comment]
> The latest-version shortcode is used to generate the API reference link in a number of files.

That is bad... 😞  Need to fix them.

**Comment by kbhawkey** [comment]
In v1.16, this is the link text:
```
[Node API object](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#node-v1-core).
```

**Comment by kbhawkey** [comment]
I think it makes sense to update the latest parameter in deprecated versions 1.19, 1.18, 1.17?
I have not found that the v1.20 string is used in the deprecated snapshots?

**Comment by raghvenders** [comment]
@kbhawkey v1.16 and below looks like have correct link references for the appropriate version.
For example : https://v1-16.docs.kubernetes.io/docs/reference/

**Comment by tengqm** [comment]
@kbhawkey It depends. If we have a requirement for all archived version to point to the *real latest*, we will need to update value of `latest` in all the archived versions whenever a new release is out. If we instead treat `latest` as a relative term, then its value should be the same as the `version` parameter in the config.toml file. I'm fine with either way.

**Comment by kbhawkey** [comment]
Holding for review
/hold

**Comment by tengqm** [comment]
> In v1.16, this is the link text:
> [Node API object](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#node-v1-core).

The implication is that `version` is the parameter to use if we want to refer to the "current" version.

File: config.toml [hunk 1]
```diff
@@ -144,7 +144,7 @@ fullversion = "v1.19.4"
 version = "v1.19"
 githubbranch = "v1.19.4"
 docsbranch = "release-1.19"
-deprecated = false
+deprecated = true
```
Comment (annajung) [inline config.toml:147]: 👍  thank you for catching this! I will add this to the docs role handbook for the future leads

**Comment by kbhawkey** [comment]
I can keep all `latest-version` updates in this PR or let each localization make the fixes.
/cc @kubernetes/sig-docs-ja-reviews @kubernetes/sig-docs-zh-reviews @kubernetes/sig-docs-ko-reviews @kubernetes/sig-docs-pt-reviews @kubernetes/sig-docs-pl-reviews @kubernetes/sig-docs-ru-reviews 

Note: I did not replace the `latest-version` in the `contribute` pages.

File: config.toml (hunk 1 above)
Comment (kbhawkey) [reply config.toml:147]: 👍 👍

**Comment by mfilocha** [comment]
>
> 
> I can keep all `latest-version` updates in this PR or let each localization make the fixes.
> /cc @kubernetes/sig-docs-ja-reviews @kubernetes/sig-docs-zh-reviews @kubernetes/sig-docs-ko-reviews @kubernetes/sig-docs-pt-reviews @kubernetes/sig-docs-pl-reviews @kubernetes/sig-docs-ru-reviews
> 
@kbhawkey No objection for including the update for Polish localization in this PR. Thanks!

**Comment by inductor** [comment]
/lgtm
/approve
from japanese side

**Pull request merged**